tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -296,6 +296,8 @@ def initial_mirror_config(self, mirror2, poolname, imagename, **kw):
             p.spawn(mirror2.create_pool, poolname=poolname)
 
         self.create_image(imagespec=imagespec, size=imagesize)
+        if kw.get("mode") == "pool" or kw.get("mirrormode") == "journal":
```

Comment:
I think it would be better if you checked the condition saying kw.get("mirrormode") != "snapshot" because when we want journal mirroring, we can either pass journal in mirrormode or we can leave it empty as well.

Reply:
updated.